wykorzystanie_formatu_paczki_archiwalnej.rst

```diff
@@ -0,0 +1,96 @@
+Wykorzystanie formatu paczki archiwalnej jako formatu danych EZD
+================================================================
+
+Wstęp
+-----
+
+W tym dokumencie opisana jest propozycja użycia tzw. formatu paczki
+archiwalnej [#paczka_archiwalna]_ jako głównego `formatu danych EZD`_.
+
+W dokumencie używane są aliasy:
+
+* "FPA" oznacza "format paczki archiwalnej"
+* "FD EZD" oznacza "format danych EZD"
+
+Zarys
+-----
+
+Elementami metryki sprawy są pojedyncze dokumenty XML, każdy w FPA.
+Treść pism przechowywana jest w oddzielnym miejscu (nazwijmy je
+"skład pism"). Elementy metryki mogą zawierać odniesienia do pism.
+
+Aby takie rozwiązanie było poprawne niezbędne są zmiany w FPA lub w
+FD EZD. Nie wszystkie zmiany podane poniżej są niezbędne.
+
+Wskazanie elementu poprzedniego metryki
+---------------------------------------
+
+W FD EZD poprzedni element wskazywany jest z użyciem URI. W FPA nie
+występuje wskazanie elementu poprzedniego jako specjalne pole do tego
+przeznaczone. W FPA jest możliwość tworzenia relacji między dokumentami.
+
+W FPA można zrealizować wskazanie poprzedniego elementu metryki za
+pomocą relacji odpowiedniego typu. Takie rozwiązanie nie jest dobre,
+ponieważ trudno wymusić dokładnie jedno wskazanie poprzedniego elementu
+metryki.
```

Comment:
Czy nie można w tym celu wykorzystać [asercyj XSD](https://www.w3.org/TR/xmlschema11-1/#cAssertions)? Dostępne są one dopiero od XML w wersji 1.1 — pytanie, na ile jest to problematyczne…